### DISCUSSION TITLE
Clarification on TechEmpower benchmarks

### DISCUSSION THREAD

**Discussion by venkatesh2090** [jooby-project/jooby#2712] [Q&A] [answered]
I was looking at TechEmpower benchmarks to find a good Java framework and found jooby.

Looking at the config, the [jooby-pgclient](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/jooby/benchmark_config.json#L96) entry is configured to display Undertow as the platform on TechEmpower.

But on inspecting the Dockerfile used, it looks like its running on [netty](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/jooby/jooby-pgclient.dockerfile#L6).

Is that a mistake or am I missing something?

**Comment by jknack** [comment, marked as answer]
Hi, 

 Seems to be a mistake in [json config](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/jooby/benchmark_config.json#L96). I didn't realize until now :(

**Reply by venkatesh2090** [reply]
Thanks for the clarification. I've opened a PR for that - https://github.com/TechEmpower/FrameworkBenchmarks/pull/7800
